packages/http-client-java/generator/http-client-generator-test/src/main/java/tsptest/armresourceprovider/models/UserAssignedIdentity.java

```diff
@@ -12,40 +12,40 @@
  */
 @Immutable
 public final class UserAssignedIdentity {
-    /*
-     * The principal ID of the assigned identity.
-     */
-    @JsonProperty(value = "principalId", access = JsonProperty.Access.WRITE_ONLY)
-    private String principalId;
-
     /*
      * The client ID of the assigned identity.
      */
     @JsonProperty(value = "clientId", access = JsonProperty.Access.WRITE_ONLY)
     private String clientId;
 
+    /*
+     * The principal ID of the assigned identity.
+     */
+    @JsonProperty(value = "principalId", access = JsonProperty.Access.WRITE_ONLY)
+    private String principalId;
+
     /**
```

Comment:
This seems just diff of order?

Reply:
Yeah, tsp changed somehow: https://github.com/Azure/typespec-azure/commit/a6ddfe2a7ac0dbe10bad40cd17508fe4862e14bb#diff-9faba230610234a913cdc72e04a77efc28670f6ff49ce035d84e4ae61113023d